mesonbuild/compilers/c.py

```diff
@@ -353,6 +353,28 @@ def __init__(self, exelist, version, for_machine: MachineChoice,
                            info, exe_wrap, **kwargs)
         MSVCCompiler.__init__(self, target)
 
+    def get_options(self):
+        opts = super().get_options()
+        c_stds = ['none', 'c89', 'c99', 'c11']
+        opts.update({
+            'std': coredata.UserComboOption(
+                'C language standard to use',
+                c_stds,
+                'none',
+            ),
+        })
+        return opts
+
+    def get_option_compile_args(self, options):
+        args = []
+        std = options['std']
+        # As of MVSC 16.7, /std:c11 is the only valid C standard option.
+        # default is c99-like, with c11 requiring /std:c11
+        # Add more as MSVC adds C standards in the future
+        if std.value in ('c11'):
```

Comment:
you should use a set, not a tuple for membership tests like this, since sets are log time, and tuples or linear time for lookups.

Reply:
More importantly, `('c11')` **is not** a tuple literal, it's a string literal wrapped in parentheses (so, equivalent to just `c11`). So this code can't work as intended. You must write `('c11',)` to produce a single-element tuple.